sdk/ml/azure-ai-ml/azure/ai/ml/operations/_evaluator_operations.py

```diff
@@ -0,0 +1,338 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=protected-access,no-value-for-parameter,disable=docstring-missing-return,docstring-missing-param,docstring-missing-rtype,ungrouped-imports,line-too-long,too-many-statements
+
+from os import PathLike
+from typing import Any, Dict, Iterable, Optional, Union, cast
+from azure.ai.ml._restclient.v2021_10_01_dataplanepreview import (
+    AzureMachineLearningWorkspaces as ServiceClient102021Dataplane,
+)
+from azure.ai.ml._restclient.v2023_08_01_preview import (
+    AzureMachineLearningWorkspaces as ServiceClient082023Preview,
+)
+from azure.ai.ml._restclient.v2023_08_01_preview.models import (
+    ListViewType,
+)
+from azure.ai.ml._scope_dependent_operations import (
+    OperationConfig,
+    OperationsContainer,
+    OperationScope,
+    _ScopeDependentOperations,
+)
+from azure.ai.ml._telemetry import ActivityType, monitor_with_activity
+from azure.ai.ml._utils._experimental import experimental
+from azure.ai.ml._utils._logger_utils import OpsLogger
+from azure.ai.ml._utils.utils import (
+    _get_evaluator_properties,
+    _is_evaluator,
+)
+from azure.ai.ml.entities._assets import Environment, Model, ModelPackage
+from azure.ai.ml.entities._assets.workspace_asset_reference import (
+    WorkspaceAssetReference,
+)
+from azure.ai.ml.exceptions import (
+    UnsupportedOperationError,
+)
+from azure.ai.ml.operations._datastore_operations import DatastoreOperations
+from azure.core.exceptions import ResourceNotFoundError
+
+from azure.ai.ml.operations._model_operations import ModelOperations
+
+ops_logger = OpsLogger(__name__)
+module_logger = ops_logger.module_logger
+
+
+@experimental
```

Comment:
If you tag the evaluators property in ml_client, tag on the class is not necessary.

Reply:
Fixed